bochs/cpu/sha.cc

```diff
@@ -256,10 +256,10 @@ void BX_CPP_AttrRegparmN(1) BX_CPU_C::SHA1RNDS4_VdqWdqIbR(bxInstruction_c *i)
     E[n+1] = D[n];
   }
 
-  op1.xmm32u(0) = A[4];
```

Comment:
This change is definitely unrelated to this PR

Reply:
Yes, it is already in master

Reply:
``git rebase`` often comes to the rescue when something has already landed into the master